snowpipe.tf

```diff
@@ -42,4 +42,8 @@ resource "snowflake_pipe" "this" {
   depends_on = [
     snowflake_stage.this
   ]
+
+  replace_triggered_by = [
+    snowflake_stage.this
```

Comment:
is there any way to also recreate when an external stage is passed in, e.g. `create_stage = false`?

Reply:
I think we can use the data source for stages and check if the stage that was input is present and then if not present recreate the pipe.

Would you be able to contribute a change where a data source: https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/data-sources/stages is fetched and then checked?

Reply:
It may take a coupla weeks for us to get to this.

Reply:
> Would you be able to contribute a change where a data source: https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/data-sources/stages is fetched and then checked?

can do, could we get the other changes in first and go from there by any chance?

Reply:
👍🏼 Ofcourse